packages/core/src/components/date-picker-legacy/readme.md

```diff
@@ -30,7 +30,7 @@ Based on the [Duet Date Picker](https://duetds.github.io/date-picker-legacy/).
 | Event           | Description                                         | Type                                                                                                                                                                               |
 | --------------- | --------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `dsoBlur`       | Event emitted the date picker input is blurred.     | `CustomEvent<{ component: "dso-date-picker-legacy"; }>`                                                                                                                            |
-| `dsoDateChange` | Event emitted when a date is selected.              | `CustomEvent<{ component: "dso-date-picker-legacy"; valueAsDate: Date \| undefined; value: string; error?: "invalid" \| "required" \| "min-range" \| "max-range" \| undefined; }>` |
+| `dsoDateChange` | Event emitted when a date is selected.              | `CustomEvent<{ component: "dso-date-picker-legacy"; valueAsDate: Date \| undefined; value: string; error?: "required" \| "min-range" \| "max-range" \| "invalid" \| undefined; }>` |
```

Comment:
Deze file jojooot een beetje heen en weer, maar als het goed is hebben we hier maandag 11-11 geen last meer van.